apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nav-sidebar/src/components/nav-item/index.tsx

```diff
@@ -38,12 +39,16 @@ const NavItem = forwardRef< HTMLLIElement, NavItemProps >(
 			item.id,
 			postType.slug
 		);
+		const trackEvent = () => {
+			recordTracksEvent( `calypso_editor_sidebar_${ postType.slug }_open` );
```

Comment:
Same comment here as in the `_add` event, maybe post type should be an event param.

Reply:
What about `calypso_editor_sidebar_item_edit`. Since "open" might mean to open the sidebar.

Reply:
Ah yes, edit much more specific!